src/common/modules/data/DefaultSettings.js

```diff
@@ -4,6 +4,9 @@
  * @module data/DefaultSettings
  */
 
+// checks for OS dark theme
+const darkTheme = window.matchMedia('(prefers-color-scheme: dark)').matches;
```

Comment:
```suggestion
const isDarkTheme = window.matchMedia('(prefers-color-scheme: dark)').matches;
```

Reply:
just push this up. couldn't do it here in Github for some reason